app/_src/gateway/kong-enterprise/partials.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Partials
+badge: enterprise
+concept_type: explanation
+---
+
+## What are Partials?
+
+Some entities in Kong share common configuration settings that often need to be repeated. For example, multiple plugins that connect to Redis may require the same connection settings. Without Partials, you would need to replicate this configuration across all plugins. If the settings change, you would need to update each plugin individually.
+
+Partials address this issue by allowing you to extract shared configurations into reusable entities that can be linked to multiple plugins. To ensure validation and consistency, Partials have defined types. The first two supported types are:
+
+- **`redis-ce`**: Stores common Redis configurations for plugins available in {{site.ce_product_name}}.
+- **`redis-ee`**: Stores common Redis configurations for plugins available in {{site.ee_product_name}}.
+
+
+## Creating a Partial
+
+Partials are a new core entity so in order to create a Partial you need to send a request to the AdminAPI like this:
```

Comment:
```suggestion
To create a Partial, send a POST request to the Admin API:
```
We don't need to talk about the entity being new, since these are evergreen docs and not a blog.